src/getTime.js

```diff
@@ -18,7 +18,12 @@
  * @returns {string} - valid time
  */
 function getTime(str) {
-  // write code here
+  const matchedStr = str.match(/\b(0[0-9]|1[0-9]|2[0-3]):[0-5][0-9]\b/);
+  if (matchedStr === null) {
+    return '';
+  }
+
+  return matchedStr[0].toString();
```

Comment:
it returns string, no need to use `toString()`
```suggestion
  return matchedStr[0];
```
![image](https://user-images.githubusercontent.com/44294945/65696183-77fd4200-e081-11e9-914b-caefcf1c898e.png)